src/base/Type.ml

```diff
@@ -62,6 +62,12 @@ module PrimType = struct
     | Bits128 -> "128"
     | Bits256 -> "256"
 
+  let int_bit_width_to_int = function
+    | Bits32 -> 32
+    | Bits64 -> 64
+    | Bits128 -> 128
+    | Bits256 -> 256
+
```

Comment:
We also have the `int_width` function further below in `Types.ml`:
```ocaml
  let int_width = function
    | PrimType (Int_typ Bits32) | PrimType (Uint_typ Bits32) -> Some 32
    | PrimType (Int_typ Bits64) | PrimType (Uint_typ Bits64) -> Some 64
    | PrimType (Int_typ Bits128) | PrimType (Uint_typ Bits128) -> Some 128
    | PrimType (Int_typ Bits256) | PrimType (Uint_typ Bits256) -> Some 256
    | _ -> None
```

Is it possible to reuse `int_bit_width_to_int` to simplify it into something like this?
```ocaml
  let int_width = function
    | PrimType (Int_typ bits) | PrimType (Uint_typ bits) -> Some (int_bit_width_to_int bits)
    | _ -> None
```

Reply:
Done